src/app/pages/applications/applications.component.ts

```diff
@@ -74,19 +74,10 @@ export class ApplicationsComponent implements OnInit, AfterViewInit {
         this.filterString = evt.data.filter;
       }
 
-      switch (this.selectedTab) {
-        case ApplicationTab.InstalledApps:
-          this.chartTab.onToolbarAction(evt);
-          break;
-        case ApplicationTab.Catalogs:
-          this.manageCatalogTab.onToolbarAction(evt);
-          break;
-        case ApplicationTab.AvailableApps:
-          this.catalogTab.onToolbarAction(evt);
-          break;
-        case ApplicationTab.DockerImages:
-          this.dockerImagesTab.onToolbarAction(evt);
-      }
+      this.chartTab.onToolbarAction(evt);
+      this.catalogTab.onToolbarAction(evt);
+      this.manageCatalogTab.onToolbarAction(evt);
+      this.dockerImagesTab.onToolbarAction(evt);
```

Comment:
I think that it should apply an event exactly into a tab content when the tab is changed so a `switch-case` condition should be keep as original code.

A `disabled` property of the **`Launch Docker Image`** button depends on `isSelectedPool` state.
So I think that the solution is to set `isSelectedPool` state correctly when the page is loaded.